subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -1439,6 +1439,12 @@ void ll_rx_mem_release(void **node_rx)
 				ull_sync_setup_complete(scan);
 
 				if (status != BT_HCI_ERR_SUCCESS) {
+					memq_link_t *link_sync_lost;
```

Comment:
Are you using local variables to comply with coding style guide?

Reply:
Mostly to match the same variable used when allocating the link: https://github.com/zephyrproject-rtos/zephyr/blob/657e2500989e97a8ceb20aa2818dd07810c6cc3f/subsys/bluetooth/controller/ll_sw/ull_sync.c#L181

Reply:
I would not create a variable just to pass it to another function and not use it again. I know compiler will optimize it but there is a code that has to be maintained. If the same approach appears in many places then the amount of such unnecessary code grows. Anyway it is a nit-pick. I'm not blocking the PR because of that. In my humble opinion we should avoid it in future.